docs/user_guide/mitigation.rst

```diff
@@ -80,11 +81,408 @@ Reductions
 
 .. currentmodule:: fairlearn.reductions
 
+From a very high-level perspective the reductions techniques within Fairlearn
+enable unfairness mitigation on an arbitrary machine learning model with
+respect to a user provided fairness objective. For more information on the
+supported kinds of fairness refer to :ref:`constraints_binary_classification`
+and :ref:`constraints_regression`. All of the constraints currently supported
+by reductions methods are group fairness constraints.
+Note that the choice of fairness constraint is crucial to the outcome, and
+choosing an unsuitable constraint can cause or worsen harms on individuals or
+groups affected by a decision. For a broader discussion of fairness as a
+sociotechnical challenge and how to view Fairlearn in this context refer to
+:ref:`fairness_in_machine_learning`.
+
+The reductions approach is based on the idea in [#2]_ to reduce binary
+classification subject to fairness constraints to a sequence of cost-sensitive
+classification problems. As a result, the reductions methods in Fairlearn
+require only black-box access to a cost-sensitive classification algorithm,
+which does not need to have any knowledge of the desired definition of
+fairness or sensitive features. [#1]_ extends this to regression scenarios.
```

Comment:
Do not start a sentence with an `[n]` reference. Author(s) & year would be OK, but we're not using those.

Reply:
done